### PR TITLE
chore: Fix formatting when editing files

### PR DESCRIPTION
**Pull request by wladiston**
Formatting was broken when we were editing files

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "tabWidth": 2,
```
Comment (erskingardner) [inline .prettierrc:2]: Can we change this to 4?

**Review by erskingardner — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (pablof7z) [reply .prettierrc:2]: yes, 2-space indent is a shitcoin 😂

**Comment by pablof7z** [comment]
ah fuck! didn't realize wlad was you, @wladiston! nice to see you here!!! 🙌

File: .prettierrc (hunk 1 above)
Comment (wladiston) [reply .prettierrc:2]: From my own defense, I copied that from another file 😉

**Comment by wladiston** [comment]
Hell yeah @pablof7z 
Glad to help 🕺 


@erskingardner I would run `format` before requesting you again to approve the files but I thing it's going to be faster for you to check the changed files now and then run the format before merging

**Review by erskingardner — APPROVED** [review]
Thanks @wladiston! Indeed, a bit easier to review this way. Looks great to me. Feel free to run `prettier` and then I'll merge.

**Comment by wladiston** [comment]
Done

**Pull request merged**